.storybook/main.js

```diff
@@ -0,0 +1,51 @@
+import { dirname, join } from "path";
+/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+const config = {
+    stories: [
+        '../packages/*/stories/*.stories.js',
+        '../tools/*/stories/*.stories.js',
+    ],
+
+    addons: [
+        getAbsolutePath("@storybook/addon-links"),
+        getAbsolutePath("@storybook/addon-essentials"),
+        getAbsolutePath("@storybook/addon-a11y"),
+        // Conditionally add the addon-designs addon temporarily
+        // https://storybook.js.org/addons/@storybook/addon-designs/
+        ...(process.env.NODE_ENV === 'development'
+            ? ['@storybook/addon-designs']
+            : []),
+        getAbsolutePath("@geometricpanda/storybook-addon-badges"),
+        '@chromatic-com/storybook'
+    ],
+
+    framework: "@storybook/web-components-vite",
```

Comment:
I noticed we don't have `rootDir`, `staticDirs`, `core`, or any `viteFinal` settings here. This is what we use in CSS:

```
	core: {
		disableTelemetry: true,
		disableWhatsNewNotifications: true,
		builder: "@storybook/builder-vite",
	},
```

Reply:
@castastrophe `disableTelemetry` and `disableWhatsNewNotifications` definitely sound good!

My understanding was that setting the framework to `@storybook/web-components-vite` caused `builder-vite` to be used by default. Is that not the case?

Reply:
I'm sure you're right, it's not explicit in the docs so I followed instructions from here as well just to be sure I leveraged the right tooling: https://storybook.js.org/docs/builders/vite

Reply:
Are there any Vite configs you think could be relevant or useful for the web components?

Reply:
Nothing is top of mind; I don’t consider myself to be an expert in either Vite or Storybook, TBH, but I think unless there are things you know of that you’d like to add, we can plan to circle back and make improvements later.